content/en/contribute/docs/style-guide.md

```diff
@@ -260,6 +261,18 @@ It is good practice for the image to link to the image file so that a larger ver
 |---|---|
 | `{{</* figure src="image.png" link="image.png" alt="Alt text" title="Image Title" */>}}` | `[![Alt text](image.png "Image Title")](image.png)` |
 
+### Videos
+
+When embedding videos, use the `youtube` shortcode. The shortcode embeds a responsive Youtube video player that only requires the video ID, e.g.:
+
+```
+https://www.youtube.com/watch?v=pFEFIY_SA7M
+```
+
+Copy the YouTube video ID that follows v= in the video’s URL and pass it to the youtube shortcode:
+
+`{{</* youtube pFEFIY_SA7M */>}}`
```

Comment:
The example structure was difficult for me to understand... here is another version based on your content:

```suggestion
When embedding videos, use the `youtube` shortcode to embed a responsive YouTube video player. 

Copy the YouTube video ID that follows `v=` in the video’s URL and pass it to the `youtube` shortcode. For instance, with `https://www.youtube.com/watch?v=pFEFIY_SA7M` the shortcode would be:

`{{</* youtube pFEFIY_SA7M */>}}`

And would display as seen here:

{{< youtube pFEFIY_SA7M >}}
```

Also, there was an [errant `e.g.`](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#latin-phrases) in there :)